### DISCUSSION TITLE
nautilus dark mode

### DISCUSSION THREAD

**Discussion by ronibastos** [i3/i3#5680] [Q&A] [answered]
Olá, alguém pode me ajudar? Sou novo no linux, uso arch linux com i3wm, gostaria de saber como colocar dark mode no nautilus? Já procurei na internet mas nada deu certo.

Hello, can anyone help me? I'm new to Linux, I use Arch Linux with i3wm, I would like to know how to put dark mode in Nautilus? I've looked online but nothing works.

**Comment by okraits** [comment]
Unfortunately, this has nothing to do with i3.

I guess, you have to change the GTK theme. Try to look under `Settings` -> `Appearance`.

**Reply by ronibastos** [reply]
tentei mas não consegui achar essa opção
I tried but couldn't find this option

**Comment by orestisfl** [comment]
See https://wiki.archlinux.org/title/GTK#Configuration_tools regarding configuring GTK themes

**Reply by ronibastos** [reply]
tentei mas não consegui executar essa opção

I tried but couldn't execute this option

**Reply by orestisfl** [reply]
For someone to help you, they would need more details in what you tried, what went wrong, what error you get.

**Reply by ronibastos** [reply]
I tried several things, and I didn't see an error message, but nothing worked, nautilus is always white.

Over a year ago, Nautilus was in dark mode, but with the last update it lost dark mode, and I can't get it to stay in dark mode.

What I had used to make it stay in dark mode was:

at the terminal:
sudo pacman -S gnome-themes-extra

then the command:

gsettings set org.gnome.desktop.interface color-scheme prefer-dark


and after the update it no longer works.

**Reply by orestisfl** [reply]
And have you tried any of the configuration tools like LXAppearance?

**Reply by ronibastos** [reply]
First of all, I would like to thank you for your willingness to help me! I don't know much about Linux and Arch and even worse about i3wm, but I'm a fan of these guys because they're very minimalist and my machine flies!! I'm very happy, and it's only now with this update that I lost dark mode in Nautilus.

I tried this solution too, I installed LXAppearance, then opened the app and set it to dark mode, saved it, opened Nautilus and it was still all white.

In the terminal, when I opened the app and made the change, it gave this message below:

(lxappearance:1458): Gtk-WARNING **: 15:29:55.833: Unable to find theming tool in module_path: "hcengine",

**Reply by ronibastos** [reply]
```
 ~]$ neofetch
                   -`              
                  .o+`                   ------------
                 `ooo/                   OS: Arch Linux x86_64
                `+oooo:                  Host: Aspire E5-571 V1.26
               `+oooooo:                 Kernel: 6.5.3-arch1-1
               -+oooooo+:                Uptime: 59 secs
             `/:-:++oooo+:               Packages: 690 (pacman)
            `/++++/+++++++:              Shell: bash 5.1.16
           `/++++++++++++++:             Resolution: 1366x768
          `/+++ooooooooooooo/`           WM: i3
         ./ooosssso++osssssso+`          Theme: Adwaita-dark [GTK2/3]
        .oossssso-````/ossssss+`         Icons: Adwaita [GTK2/3]
       -osssssso.      :ssssssso.        Terminal: alacritty
      :osssssss/        osssso+++.       CPU: Intel i5-5200U (4) @ 2.700GHz
     /ossssssss/        +ssssooo/-       GPU: Intel HD Graphics 5500
   `/ossssso+/:-        -:/+osssso+-     Memory: 376MiB / 5844MiB
  `+sso+:-`                 `.-/+oso:
 `++:.                           `-/+/
 .`                                 `/
```

**Reply by orestisfl** [reply, marked as answer]
Ok nautilus uses gtk4. It is supposed to work with
```
gsettings set org.gnome.desktop.interface color-scheme prefer-dark
```
but I couldn't get it to work locally

However, a quick workaround is to run:
```
GTK_THEME=Adwaita:dark nautilus
```

which does work on my machine

**Reply by ronibastos** [reply]
This one worked! GTK_THEME=Adwaita:dark nautilus

I put it as a shortcut in my configuration file and it worked perfect!

Thank you very much!!!

**Comment by ronibastos** [comment]
I discovered this solution below with the help of a member of the Arch forum:

Gnome Shell's dark mode is controlled by the libadwaita package, and version 1:1.4.0-1 broke this feature on my system.

The problem was the lack of a libadwaita dependency, which is xdg-desktop-portal.

sudo pacman -S xdg-desktop-portal

This solution also solved it! And I would like to share it too!
